tests/end_to_end/end_to_end.cpp

```diff
@@ -2685,3 +2685,121 @@ TEST_CASE("test_ebpf_object_set_execution_type", "[end_to_end]")
 
     bpf_object__close(jit_object);
 }
+
+static void
+extension_reload_test(ebpf_execution_type_t execution_type)
+{
+    _test_helper_end_to_end test_helper;
+    // Create a 0-byte UDP packet.
+    auto packet0 = prepare_udp_packet(0, ETHERNET_TYPE_IPV4);
+
+    // Test that we drop the packet and increment the map
+    xdp_md_t ctx0{packet0.data(), packet0.data() + packet0.size(), 0, TEST_IFINDEX};
+
+    // Try loading with out the extension loaded.
```

Comment:
```suggestion
    // Try loading without the extension loaded.
```

Reply:
As it exists today, the hash is computed prior to knowing which helpers functions the program uses. This forces any change to the program information to require a reverification.